rule-packs/mitre-attck-lateral-movement-attack-paths.json

```diff
@@ -0,0 +1,626 @@
+        "alertLevel": "HIGH"
+      },
+      {
+        "name": "remote-services-smb-windows-admin-shares-password-anti-reuse",
+        "description": "M1027 - Password Policies - Do not reuse local administrator account passwords across systems. Ensure password complexity and uniqueness such that the passwords cannot be cracked or guessed.",
+        "queries": [
+          {
+            "name": "query0",
+            "query": "find aws_iam_account_password_policy with historyCount < 10 or historyCount=undefined",
+            "version": "v1"
+          }
+        ],
+        "alertLevel": "HIGH"
+      },
+      {
+        "name": "remote-services-vnc-restrict-vnc-traffic-port-5900",
+        "description": "M1037 - Filter Network Traffic - VNC defaults to TCP ports 5900 for the server, 5800 for browser access, and 5500 for a viewer in listening mode. Filtering or blocking these ports will inhibit VNC traffic utilizing default ports.",
+        "queries": [
+          {
+            "name": "query0",
+            "query": "FIND Firewall AS fw THAT ALLOWS AS rule * AS src WHERE rule.ingress = true AND rule.ipProtocol = 'vnc' AND rule.fromPort <= 5900 AND rule.toPort >= 5900",
+            "version": "v1"
+          }
+        ],
+        "alertLevel": "HIGH"
+      },
+      {
+        "name": "remote-services-vnc-restrict-vnc-traffic-port-5800",
+        "description": "M1037 - Filter Network Traffic - VNC defaults to TCP ports 5900 for the server, 5800 for browser access, and 5500 for a viewer in listening mode. Filtering or blocking these ports will inhibit VNC traffic utilizing default ports.",
+        "queries": [
+          {
+            "name": "query0",
+            "query": "FIND Firewall AS fw THAT ALLOWS AS rule * AS src WHERE rule.ingress = true AND rule.ipProtocol = 'vnc' AND rule.fromPort <= 5800 AND rule.toPort >= 5800",
+            "version": "v1"
+          }
+        ],
+        "alertLevel": "HIGH"
+      },
+      {
+        "name": "remote-services-vnc-restrict-vnc-traffic-port-5500",
+        "description": "M1037 - Filter Network Traffic - VNC defaults to TCP ports 5900 for the server, 5800 for browser access, and 5500 for a viewer in listening mode. Filtering or blocking these ports will inhibit VNC traffic utilizing default ports.",
+        "queries": [
+          {
+            "name": "query0",
+            "query": "FIND Firewall AS fw THAT ALLOWS AS rule * AS src WHERE rule.ingress = true AND rule.ipProtocol = 'vnc' AND rule.fromPort <= 5500 AND rule.toPort >= 5500",
+            "version": "v1"
+          }
+        ],
+        "alertLevel": "HIGH"
+      },
+      {
+        "name": "remote-services-cloud-services-use-mfa-aws",
+        "description": "M1032 - Multi-factor Authenticaion - Use multi-factor authentication on remote service logons where possible in aws.",
+        "queries": [
+          {
+            "name": "query0",
+            "query": "FIND aws_iam_user WITH mfaEnabled!=true",
```

Comment:
duplicate rule as above

Reply:
Not duplicate, different subtechniques. 
remote-services-use-mfa-aws
vs
remote-services-cloud-services-use-mfa-aws